processes/security_team_members.md

```diff
@@ -7,94 +7,74 @@ yet been disclosed publicly, including the existence of issues, expectations of
 upcoming releases, and patching of any issues other than in the process of their
 work as a member of the security team.
 
-Membership on the security teams can be requested via an issue in the TSC repo,
-and must be approved by current team members.
+## Node.js Security Team Membership Policy
 
-Members of the security teams should indicate that they accept the privacy
-policies by PRing their acceptance to this file.
+The Node.js Security Team has access to security-sensitive issues and patches
+that aren't appropriate for public availability.
 
-## Team that triages security reports against node core
+The policy for inclusion is as follows:
 
-- @bnoordhuis - **Ben Noordhuis**
-- @cjihrig - **Colin Ihrig**
-- @indutny - **Fedor Indutny**
-- @jasnell - **James M Snell**
-- @mcollina - **Matteo Colina**
-- @mhdawson - **Michael Dawson**
-- @MylesBorins - **Myles Borins**
-- @rvagg - **Rod Vagg**
-- @vdeturckheim - **Vladimir de Turckheim**
+1. All members of @nodejs/TSC have access to private security reports and
+   private patches.
+2. Members of the [release team](https://github.com/nodejs/node#release-team)
+   have access to private security patches in order to produce releases.
+3. On a case-by-case basis, individuals outside the Technical Steering
+   Committee are invited by the TSC to have access to private security reports
+   or private patches so that their expertise can be applied to an issue or
+   patch. This access may be temporary or permanent, as decided by the TSC.
 
-### Emeritus
+Membership on the security teams can be requested via an issue in the TSC repo.
 
-- @jasnell - **James M Snell**
-- @shigeki - **Shigeki Ohtsu**
+## Team that triages security reports against Node.js
 
-List is from ["security" alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+The [TSC](https://github.com/nodejs/node#tsc-technical-steering-committee)
```

Comment:
I don't believe this is accurate.  All TSC members have access  to info about discussions/patches related to security vulnerabilities but that does not mean they are part of the team that has volunteered to do the triage.